docs/standard/frameworks.md

```diff
@@ -101,11 +78,49 @@ A framework is typically referenced by a short TFM. A TFM value can represent on
 | Universal Windows Platform | uap          | uap [uap10.0]                                |
 |                            |              | uap10.0 [win10] [netcore50]                  |
 
-## Deprecated frameworks
+## How to specify target frameworks
+
+Target frameworks are specified in your *csproj* project file. When a single target framework is specified, use the **\<TargetFramework>** element. The following console app project file demonstrates how to target the `netcoreapp1.0` TFM:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
 
-The following frameworks are deprecated. Packages targeting these frameworks should migrate to the indicated replacements.
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+  </PropertyGroup>
 
-| Deprecated framework | Replacement |
+</Project>
+```
+
+The following library project file targets newer APIs of .NET Standard (`netstandard1.4`) and older APIs of the .NET Framework (`net40` and `net45`). When targeting more than one target framework, use the plural **\<TargetFrameworks>** element. Note how the use of `Condition` attributes includes implementation-specific package references when the library is compiled for the two .NET Framework TFMs:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.4;net40;net45</TargetFrameworks>
+  </PropertyGroup>
+
+  <!-- Conditionally obtain references for the .NET Framework 4.0 target -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net40' ">
+    <Reference Include="System.Net" />
+  </ItemGroup>
+
+  <!-- Conditionally obtain references for the .NET Framework 4.5 target -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Threading.Tasks" />
+  </ItemGroup>
+
+</Project>
+```
+
+## Deprecated target frameworks
+
+The following target frameworks are deprecated. Packages targeting these frameworks should migrate to the indicated replacements.
+
+| Deprecated TFM       | Replacement |
 | -------------------- | ----------- |
 | aspnet50             | netcoreapp  |
 | aspnetcore50         |             |
```

Comment:
I assume the empty cells under netcoreapp implicitly contain netcoreapp also; might be clearer to make that explicit